### PR TITLE
Refactor RadialBarSeries animations to use `fromToMotion`.

### PR DESCRIPTION
**Pull request by alantreadway**
Bonus:
- Improve bar-like cases.

### CODE REVIEW

File: packages/ag-charts-enterprise/src/series/radial-bar/__image_snapshots__/__diff_output__/radial-bar-test-ts-radial-bar-chart-initial-animation-for-example-options-should-animate-at-25-1-snap-diff.png
Comment (lsjroberts) [inline packages/ag-charts-enterprise/src/series/radial-bar/__image_snapshots__/__diff_output__/radial-bar-test-ts-radial-bar-chart-initial-animation-for-example-options-should-animate-at-25-1-snap-diff.png:1]: 👀

**Review by lsjroberts — CHANGES_REQUESTED** [review]
some diffs to remove

File: packages/ag-charts-enterprise/src/series/radial-bar/__image_snapshots__/__diff_output__/radial-bar-test-ts-radial-bar-chart-initial-animation-for-example-options-should-animate-at-25-1-snap-diff.png
Comment (alantreadway) [reply packages/ag-charts-enterprise/src/series/radial-bar/__image_snapshots__/__diff_output__/radial-bar-test-ts-radial-bar-chart-initial-animation-for-example-options-should-animate-at-25-1-snap-diff.png:1]: Fixed:
1) Reason for diff is that there was no easing function previously.
2) I've removed the diffs here - apologies.

**Review by lsjroberts — APPROVED** [review]

**Comment by alexanderby** [comment]
Screenshot 2023-10-04 at 11 20 44
If it's linear easing, the shapes shouldn't grow over halp of the circle at 50%.

**Pull request merged**